sqlmesh/core/macros.py

```diff
@@ -866,6 +876,17 @@ def or_(evaluator: MacroEvaluator, *expressions: t.Optional[exp.Expression]) ->
     return exp.or_(*conditions, dialect=evaluator.dialect)
 
 
+@macro("VAR")
+def var(
+    evaluator: MacroEvaluator, var_name: exp.Expression, default: exp.Expression = exp.null()
```

Comment:
default should be optional null, python doesn't allow defaults types here because they are mutable and singletons

default = None
default = exp.null() if default is None

Reply:
I can see us doing exactly this here: https://github.com/TobikoData/sqlmesh/blob/main/sqlmesh/core/macros.py#L822-L823. What's the difference?

Reply:
that's a bug